modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/view/table/DateTimeFDSTableSchemaField.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.data.set.view.table;
+
+/**
+ * @author Guilherme Camacho
+ */
+public class DateTimeFDSTableSchemaField extends BaseDateFDSTableSchemaField {
+
+	@Override
+	public String getContentRenderer() {
+		return "datetime";
```

Comment:
Hey @markocikos , we had to change it to "dateTime" as requested by Brian. He merged it some hours ago [here](https://github.com/brianchandotcom/liferay-portal/pull/111367), please do a rebase when synced upstream. In the meantime @marcelabc or @gabrielwas can review the other parts of your code. Thanks a lot!